pkg/adaptor/kinesis/kinesis.go

```diff
@@ -0,0 +1,177 @@
+package adaptor
```

Comment:
change this to `package kinesis` and then import `"github.com/compose/transporter/pkg/adaptor"`